src/test/java/com/commercetools/sync/services/impl/CartDiscountServiceImplTest.java

```diff
@@ -31,6 +45,12 @@
 
     private String testCartDiscountKey = "ABC123";
 
+    private static final String PREDICATE_1 = "1 = 1";
```

Comment:
seems like these values are only used in one test, then it's better to inline them where they are used.

Reply:
6f256ee

Reply:
even better!